doc/source/whatsnew/v1.1.0.rst

```diff
@@ -434,7 +434,8 @@ Performance improvements
 
 Bug fixes
 ~~~~~~~~~
-
+Fixed Series.value_counts so that normalize excludes NA values when dropna=False. (:issue:`25970`)
```

Comment:
can you move this down into the Numeric section. starts at L482

Reply:
Done